src/get-files-stream.js

```diff
@@ -82,7 +82,8 @@ function loadPaths (opts, file) {
         // files inside symlinks and others
         return
       })
-      .filter((file) => !!file) // filter out null files
+      // filter out null files
+      .filter((f) => Boolean(f))
```

Comment:
Oh nice, didn't know about this wrapper. We can make it just `.filter(Boolean)`, right?